js/calculation.js

```diff
@@ -0,0 +1,375 @@
+//declare global variables
+    var w = document.getElementById('weight');
+    var a = window.document.getElementById('age');
+    var h = window.document.getElementById('height');
+    var c = window.document.getElementById('creatinine');
+    var m = document.getElementById("male");
+    var f = document.getElementById("female");
+    var y = document.getElementById("yes");
+
+    
+// this function displays the form when the checkbox is true
+    var checkbox = document.getElementById('showform');
+    function displayForm() {
+        var vancform = document.getElementById('vancform');
+        
+        if (checkbox.checked) {
+            vancform.style.display = 'block';
+        } else {
+            vancform.style.display = 'none';
+        }
+    }
+    
+    //this function hides the parameters to make more space for the form if slected
+    function hideParameters() {
+        var parameters = document.getElementById('parameters');
+        
+         if (checkbox.checked) {
+            parameters.style.display = 'none';
+        } else {
+            parameters.style.display = 'block';
+        }
+    }
+    
+    
+    function showGuideline() {
+        var guideline = document.getElementById('guideline');
+            guideline.style.display = 'block';
+    }
+ 
+
+    function displayIndication() {
+        var ind = document.getElementById('ind').value;
+        return document.getElementById('indication').innerHTML = "The indication for treatment is " + "<b>" + ind + "<b>";
+    }
+    
+    // function CRRTform() {
+    //     var shortForm
+    // shortForm = document.getElementById("height").style.display= 'none';
+    // shortForm = document.getElementById("datepicker").style.display= 'none';
+    // shortForm = document.getElementById("age").style.display= 'none';
+    // shortForm = document.getElementById("creatinine").style.display= 'none';
+    // }
+    
+    //Sets date picker max date to today
+    var today = new Date();
+    var dd = today.getDate();
+    var mm = today.getMonth()+1; //January is 0!
+    var yyyy = today.getFullYear();
+    
+    if(dd<10){
+        dd='0'+dd
+    } 
+    if(mm<10){
+        mm='0'+mm
+    } 
+
+    today = yyyy+'-'+mm+'-'+dd;
+    document.getElementById("datepicker").setAttribute("max", today);
+    
+    //Sets date picker min date to minus 30 days
+    var today = new Date();
```

Comment:
you gave two variables the same name `today`... you don't need this code. Consider creating a function that returns two dates.
```
function getMinMaxDates(){
   var today = new Date();
    var dd = today.getDate();
    var mm = today.getMonth()+1; //January is 0!
    var yyyy = today.getFullYear();
    if(dd<10){
        dd='0'+dd
    } 
    if(mm<10){
        mm='0'+mm
    } 
   let maxDate = yyyy+'-'+mm+'-'+dd;
   var mm1 = today.getMonth();
    if(dd<10){
        dd='0'+dd
    } 
    if(mm1<10){
        mm1='0'+mm1
    } 
    let minDate = yyyy+'-'+mm1+'-'+dd;
    return [maxDate, minDate]
}
```

// Then you use array destructing to get the dates outside the function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring
`let [maxDate, minDate] = getMinMaxDates()`
// Then set your attributes
```
var datepicker = document.getElementById("datepicker")
datepicker.setAttribute("max", maxDate);
datepicker.setAttribute("min", minDate)

```